gw_eccentricity/gw_eccentricity.py

```diff
@@ -1,4 +1,4 @@
-"""gw_eccentricity.
+"""gw_eccentricity wrapper function to call different eccDefinition methods.
```

Comment:
Given that there are multiple functions, get rid of "function"

Reply:
Done